newsfragments/template.significant.rst

```diff
@@ -3,20 +3,29 @@
 .. Provide additional contextual information
 
 .. Check the type of change that applies to this change
+.. Dag changes: requires users to change their dag code
+.. Config changes: requires users to change their airflow config
+.. API changes: requires users to change their Airflow REST API calls
+.. API changes: requires users to change their Airflow usage
```

Comment:
Here is a small Uuups, I assume the second shouldbe CLI?
```suggestion
.. API changes: requires users to change their Airflow REST API calls
.. CLI changes: requires users to change their Airflow usage
```

Reply:
Ah yes and I miss CLI in the description as well 🤦‍♂️

Reply:
fixed!